### ISSUE TITLE
[Feature] Allow Floating all windows by Default

### ISSUE THREAD

**Issue by RicardoEPRodrigues** [pop-os/shell#898]
Although it might go against the whole purpose of the extension, I would like to have an option that would make any window Floating by default.

This would mean that no tiling will occur until I tile the window (Ctrl + G), which is exactly what I want. As my workflow tends to require more screen real-estate and not allow me to use tiling for everything, but I would still like to tile certain windows.

I looked around the issues and there doesn't seem to be a feature request like this yet, but correct me if I'm wrong. Thank you!

**Comment by mmstick** [comment]
You could use workspaces for applications that need more screen space.

**Comment by RicardoEPRodrigues** [comment]
True, but that does not fit my workflow, and some applications (such as games) break when tiling messes with their size.
